skll/utilities/compute_eval_from_predictions.py

```diff
@@ -10,25 +10,85 @@
 import argparse
 import csv
 import logging
+import random
 
 from skll.data import Reader, safe_float
 from skll.metrics import use_score_func
 from skll.version import __version__
 
 
+def get_prediction_from_probabilities(classes, probs, prediction_method):
+    """
+    Convert a list of class-probabilities into a class prediction. This function assumes
+    that, if the prediction method is 'expected_value', the class labels are integers.
+
+    Parameters
+    ----------
+    classes: list
+        List of str or int class names.
+
+    probs: list of float
+        Probabilities for respective classes.
+
+    prediction_method: str
+        Indicates how to get a single class prediction from the probabilities. Currently
+        supported options are  "highest", which selects the class with the highest
+        probability, and "expected_value", which calculates an expected value over
+        integer classes and rounds to the nearest int.
+
+    Returns
+    -------
+    str or int
+        Predicted class.
+
+    """
+    if prediction_method == 'highest':
+        if len(set(probs)) == 1:  # probabilities are all equal
+            return random.choice(classes)
+        else:
+            return classes[probs.index(max(probs))]
+    elif prediction_method == 'expected_value':
+        exp_val = sum([classes[i] * prob for i, prob in enumerate(probs)])
+        return int(round(exp_val))
+
+
 def compute_eval_from_predictions(examples_file, predictions_file,
```

Comment:
While we are adding a docstrings, can we fix the arguments to be one per line properly indented?